app/content/models/registration.py

```diff
@@ -29,6 +31,71 @@ class Meta:
         verbose_name = "Registration"
         verbose_name_plural = "Registrations"
 
+    @staticmethod
```

Comment:
I see there are more specific permissions for this model, how about specifying fields like `detail_write_access`, `write_access` and so on?